### ISSUE TITLE
Insert/Remove HTML markup in header during sort

### ISSUE THREAD

**Issue by harisb** [Mottie/tablesorter#1178]
I am trying to have sort icons created via font awesome, in other words, I need to add and remove additional markup based on sort ASC or DESC. 

I saw these two doc pages, but it doesn't look like they'll do exactly what I want.  Or I'm not getting how it works exactly.

http://mottie.github.io/tablesorter/docs/example-option-render-header.html
http://mottie.github.io/tablesorter/docs/example-option-render-template.html

I do know about changing the classes of headers, but that won't work because that applies the icon font to non icon characters too.

Here's an example of markup to insert a font awesome icon:

`<i class="fa fa-sort-amount-asc"></i>`

So in other words, I would need it to look like this upon sort:

`<th><i class="fa fa-sort-amount-asc"></i> Column Name</th>`

or

`<th>Column Name <i class="fa fa-sort-amount-asc"></i></th>`

Is something like this possible with the current code?

**Comment by Mottie** [comment]
Hi @harisb!

Yes you can ([demo](http://jsfiddle.net/Mottie/856bzzeL/943/))!

First you set up the [`headerTemplate` option](http://mottie.github.io/tablesorter/docs/#headertemplate) to add an icon...

* It will automatically add an `<i>` when you set it to `{content}{icon}`. The `<i>` will get the class name set in the [`cssIcon` setting](http://mottie.github.io/tablesorter/docs/#cssicon).

    ```js
    headerTemplate: '{content} {icon}'
    ```

* Or you can add custom HTML, and again, the class name needs to match the [`cssIcon` setting](http://mottie.github.io/tablesorter/docs/#cssicon).

    ```js
    headerTemplate: '{content} <i class="tablesorter-icon fa"></i>'
    ```

Then set the `cssIcon`, [`cssIconNone`](http://mottie.github.io/tablesorter/docs/#cssiconnone), [`cssIconAsc`](http://mottie.github.io/tablesorter/docs/#cssiconasc) and [`cssIconDesc`](http://mottie.github.io/tablesorter/docs/#cssicondesc) options

```js
cssIcon     : 'fa',
cssIconAsc  : 'fa-sort-amount-asc',
cssIconDesc : 'fa-sort-amount-desc',
cssIconNone : 'fa-sort'
```

If you're using a custom theme and the filter widget, make sure to include this definition:

```css
.tablesorter .filtered {
    display: none;
}
```

\* Note \* The `filtered` class is set by the [`filter_filteredRow` option](http://mottie.github.io/tablesorter/docs/#widget-filter-filteredrow) .

**Comment by harisb** [comment]
Awesome, works great. Thanks @Mottie
